uk_bin_collection/uk_bin_collection/councils/SouthDerbyshireDistrictCouncil.py

```
@@ -4,57 +4,40 @@
         user_uprn = kwargs.get("uprn")
         check_uprn(user_uprn)
         data = {"bins": []}
 
         baseurl = "https://maps.southderbyshire.gov.uk/iShareLIVE.web//getdata.aspx?RequestType=LocalInfo&ms=mapsources/MyHouse&format=JSONP&group=Recycling%20Bins%20and%20Waste|Next%20Bin%20Collections&uid="
         url = baseurl + user_uprn
 
-        # Make the web request
-        response = requests.get(url).text
+        requests.packages.urllib3.disable_warnings()
+        headers = {"User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36"}
+        response = requests.get(url, verify=False, headers=headers).text
```

Ruff (0.15.12)

[error] Probable use of `requests` call without timeout

(S113)